.github/workflows/translations-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync translations
```

Comment:
This workflow will:
1. Upload new translation strings to Crowdin
2. Download any new approved translations from Crowdin